src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -190,8 +189,9 @@ def _load_simulations(runpath: str, ecl_base: str) -> EclSum:
     Returns:
         EclSum
     """
-
-    return EclSum(str(pathlib.Path(runpath) / pathlib.Path(ecl_base)))
+    if os.path.exists(pathlib.Path(runpath) / pathlib.Path(ecl_base + ".UNSMRY")):
+        return EclSum(str(pathlib.Path(runpath) / pathlib.Path(ecl_base)))
+    return None
```

Comment:
You are allowing the function to return a `None`. This means you will have to update the documentation and the type annotation to reflect this.

`def _load_simulations(runpath: str, ecl_base: str) -> Optional[EclSum]:`